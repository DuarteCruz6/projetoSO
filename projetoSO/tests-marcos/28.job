WRITE [(z5,10)]
READ [z5]
WRITE [(p1,68)]
WRITE [(p8,47)(w8,10)]
WRITE [(z8,2)(p5,5)(s0,0)]
WRITE [(y1,36)(b6,33)(u5,53)]
DELETE [z8,b6]
WRITE [(h6,49)(b0,6)]
READ [u5]
WAIT 1000
WRITE [(w3,63)]
WRITE [(t9,86)]
READ [t9]
SHOW
WRITE [(b1,100)(c3,76)(n3,9)(c7,58)]
READ [p1]
READ [c7]
READ [u5]
WRITE [(j3,76)(t0,99)(h2,12)]
DELETE [z5,y1,n3,j3,h2]
READ [s0]
WAIT 1000
WRITE [(c0,77)]
READ [b0]
DELETE [b0,w3,p8,p5]
DELETE [p1,w8]
SHOW
WRITE [(z4,77)(y9,53)(b7,88)]
WRITE [(r4,42)(l3,87)(l1,15)(x2,62)]
DELETE [c7]
DELETE [y9,c3,b1,b7]
WRITE [(i6,93)(n3,17)(d1,99)(s8,17)]
WRITE [(m2,96)(s9,64)]
WRITE [(f1,13)(n0,56)(c1,2)(t3,22)]
BACKUP
WRITE [(q4,4)]
READ [s8]
WRITE [(p5,9)]
WAIT 3000
DELETE [f1,u5]
WRITE [(j6,44)]
WRITE [(s3,46)(w0,79)]
READ [c1]
WRITE [(f8,48)(y9,29)(q2,81)]
READ [s0]
SHOW
WRITE [(g7,21)(f3,30)(a9,71)]
WRITE [(v1,27)(y4,89)]
READ [c1]
WRITE [(n4,89)]
BACKUP
READ [q2]
WRITE [(h1,89)(j3,32)(l6,63)(a6,12)]
READ [s9]
WRITE [(d0,75)(b9,2)]
READ [m2]
READ [q4]
WRITE [(s2,19)(s1,6)(c7,99)(y5,92)(g1,42)]
SHOW
WRITE [(r0,0)(j2,91)(u9,92)]
SHOW
WRITE [(c2,90)(x3,25)(x1,95)]
WRITE [(v3,47)(s4,5)(u0,17)(u6,33)]
DELETE [j3,h6,y5]
WRITE [(t6,48)(h7,98)(m1,4)(j5,22)(b5,7)]
WRITE [(p8,82)(j7,41)(x5,83)]
READ [y9]
SHOW
READ [j6]
WRITE [(p1,20)(e4,76)(y7,88)]
WRITE [(p7,2)(o6,15)]
WRITE [(q1,92)(m3,75)(q9,23)(r2,56)]
DELETE [s8]
WRITE [(i5,25)(c5,50)(y8,88)(t1,85)(f5,55)]
WRITE [(u5,59)]
DELETE [y9,i5]
WRITE [(p0,57)(s7,25)(w6,96)(t7,64)(o9,31)]
WRITE [(c8,0)(s6,93)(r8,49)]
DELETE [t3,x3,m3]
DELETE [r8,p0]
READ [t7]
READ [e4]
WRITE [(i2,8)(l8,86)(a5,93)(j9,0)]
WRITE [(f4,29)(l4,13)(a2,25)(h8,24)]
READ [p8]
SHOW
READ [s7]
SHOW
