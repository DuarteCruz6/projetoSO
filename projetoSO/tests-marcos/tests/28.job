WRITE [(o8,96)(t9,91)(f2,12)(l9,86)(b7,96)]
READ [o8]
BACKUP
WRITE [(n2,71)(k5,98)(k0,24)]
WRITE [(a7,99)]
READ [t9]
DELETE [b7,o8]
READ [t9]
DELETE [l9,f2,k0,a7,n2]
DELETE [k5,t9]
WRITE [(f9,54)(z3,63)]
READ [z3]
DELETE [f9,z3]
WRITE [(v6,83)(u6,71)(l8,23)(z8,47)(q4,11)]
SHOW
READ [l8]
BACKUP
WRITE [(k1,37)(d4,13)(a3,80)(s4,17)]
DELETE [a3]
READ [k1]
READ [z8]
BACKUP
WRITE [(l7,80)]
WRITE [(l4,24)(x2,98)(h1,91)(c5,23)(g5,20)]
WRITE [(n4,92)(k4,11)(u1,60)(h5,80)]
BACKUP
DELETE [z8,k1,v6,x2]
WRITE [(m3,86)]
WRITE [(v4,43)(j1,14)]
WRITE [(n0,6)(h4,64)(x4,66)(q2,10)(v6,17)]
WRITE [(c7,96)(i0,45)(v0,1)(c8,72)]
WAIT 2000
WRITE [(y1,87)(v1,72)]
WRITE [(g7,40)(z0,69)(h8,25)(o8,83)(f3,34)]
READ [c8]
WAIT 1000
DELETE [l4,s4,y1]
WRITE [(f0,7)(w7,11)(r3,97)(b7,99)]
DELETE [v4,l8,h8,v0]
WRITE [(r8,71)]
WRITE [(l9,76)(u3,53)]
WRITE [(t3,48)(x6,73)(s7,8)(f9,37)(x8,80)]
WRITE [(w6,23)(p4,37)(d6,49)(e9,29)]
WRITE [(a5,80)(l3,35)(m5,7)(f2,26)(x0,62)]
BACKUP
WRITE [(b4,89)(v2,82)(o5,100)]
WRITE [(e5,86)(j5,34)]
