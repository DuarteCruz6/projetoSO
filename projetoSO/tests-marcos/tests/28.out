[(o8,96)]
[(t9,91)]
[(t9,91)]
[(z3,63)]
(l8, 23)
(q4, 11)
(u6, 71)
(v6, 83)
(z8, 47)
[(l8,23)]
[(k1,37)]
[(z8,47)]
[(c8,72)]
